tests/functional/test_override_ssl_common_name.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from awscli.testutils import BaseAWSCommandParamsTest
+from awscli.customizations.overridesslcommonname import SSL_COMMON_NAMES
+
+
+class OverrideSSLCommonNameTestCase(BaseAWSCommandParamsTest):
+    def _common_name_test_cases(self):
+
+        service_ops = {
+            "sqs": "list-queues",
+            "emr": "list-clusters",
+            "rds": "describe-db-clusters",
+        }
+        for service, regions in SSL_COMMON_NAMES.items():
+            for region in regions:
+                yield (service, service_ops[service], region)
```

Comment:
probably doesn't matter for this. but i got a warning from `pytest`:
```bash
PytestCollectionWarning: yield tests were removed in pytest 4.0 - test_cases will be ignored
```

Reply:
I wonder if it is the name `_common_name_test_cases`  that is triggering it? Like if we remove `test` from the method. It would be nice to not have the warning to make the py.test output cleaner.

Reply:
yup that did it! good call